TopMovies/View/MovieVC/MovieVCProps.swift

```diff
@@ -0,0 +1,65 @@
+//
+//  MovieVCProps.swift
+//  TopMovies
+//
+//  Created by anikolaenko on 25.03.2021.
+//
+
+import UIKit
+
+struct MovieVCProps {
+  let posterPlaceholderImage: UIImage
+  let isDownloadingInProgress: Bool
+  let isAdultImage: UIImage?
+  let isNewImage: UIImage?
```

Comment:
`is` prefix usually applied for Boolean values, so you'd either rename those properties, or split them on `isNewImageShown: Bool` and `newImage: UIImage`.